ffc-pay-alerts/notify/notify-template/get-template-id.js

```diff
@@ -0,0 +1,9 @@
+const templateSchema = require('../../schema/event-schema')
+const { notifyEmailTemplateId } = require('../../config')
+
+const getTemplateId = (message) => {

```

Comment:
can you add `unit` tests for this function, testing when message is:

- message for each of the 5 `name` keys
- message wtih an  each of the 5 `name` keys
- {}
- ''
- true
- false
- undefined
- null
- 1
- 0

Reply:
Test cases added